Destiny-Corporation/Maremansa-kids
# Added persistent login functionality

**Description:**

This pull request adds persistent login functionality so that users remain logged in even after closing the browser or refreshing the page.

**Changes:**

Implemented authenticated user storage in localStorage to enable persistent login.
Updated the user state based on localStorage when loading the page.

**Steps to Test:**

- Enter a valid email and password on the login page.
- Click the "Enter" button.
- Check if you are successfully redirected to the protected page.
- Close the browser and open it again.
- Check whether you remain logged in automatically after reopening your browser.

**Additional Observations:**

This pull request aims to improve user experience during the login process and increase security by keeping users persistently authenticated.
